app/build.gradle

```diff
@@ -213,13 +213,13 @@ android {
     }
 
     buildTypes {
-//        gitUpdateModules()
-//        proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-
         release {
             signingConfig signingConfigs.release
-            minifyEnabled = false
+            minifyEnabled = true
```

Comment:
What's minifyEnabled?

Reply:
This is a feature AS does to reduce the size of the binary by compressing the libraries